### PR TITLE
Nt/challenge

### PR DESCRIPTION
**Pull request by ntaboada**
# Description

Updated README with the new Goals of the challenge and some basic instructions
Refactor from "var" to "const" syntax.
Updated package.json
Updated .gitignore

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Test A
- [ ] Test B

# Checklist:

- [ ] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,20 +28,23 @@ The datamodel can be found at `./database/datamodel.graphql`. By the end of the
...
 
-- [ ] Update and Delete functionality
+- [ ] CRUD functionality for WineTasters
+- [ ] Include a small Login Component to start supporting Users in the app.
 - [ ] Review expanded with fields such as: predicted price, suggested pairing, or predicted year
-- [ ] WineTaster expanded to include other user profile fields
-- [ ] Wines have an attached image (a placeholder image is found in `./src/assets`)
-- [ ] TastingSessions have a date
+- [ ] Include another section in the app to search reviews and filter them based on different parameters (such as best/worst rated, date, price).
+- [ ] Support the ability to use enviroment variables. Both in the Front and Back of the app.
```
Comment (justinkaseman) [inline README.md]: 😉
```suggestion
- [ ] Support the ability to use environment variables. Both in the Front and Back of the app.
```

File: README.md [hunk 2]
```diff
@@ -1,19 +1,21 @@
...
 
-Now, for the actual challenge. Assume there's exists a service that can callback in the form of a webhook when new data becomes available. These webhooks will callback with an image of a wine bottle/label. As such, we must gather data from the image to add new content to the database. For the purposes of this challenge, the candidate can use whatever images they choose. We recommend those from the [LCBO](http://www.lcbo.com/content/lcbo/en.html#.XEpE6M9KjUY) website. In any case, tests can be written with mock images to verify the correctness of the solution. 
+Now, for the actual challenge. Assume there's exists a service that can callback in the form of a webhook when new data becomes available. These webhooks will callback with an image of a wine bottle/label. As such, we must gather data from the image to add new content to the database. For the purposes of this challenge, the candidate can use whatever images they choose. We recommend those from the [LCBO](http://www.lcbo.com/content/lcbo/en.html#.XEpE6M9KjUY) website. In any case, tests can be written with mock images to verify the correctness of the solution.
```
Comment (danaszova) [inline README.md:14]: When we discussed together, you mentioned that there was some progress made to the back end? For example - a rest endpoint was set up to access the images wasn't it and Tesaract was installed. Can we give some instructions here to explain how to move this work forward?

File: package.json [hunk 3]
```diff
@@ -24,5 +26,8 @@
     "not dead",
     "not ie <= 11",
     "not op_mini all"
-  ]
+  ],
+  "devDependencies": {
+    "sweetalert2": "^8.2.6"
```
Comment (justinkaseman) [inline package.json]: Wouldn't this also need to be a regular dependency?

File: README.md [hunk 4]
```diff
@@ -26,20 +28,23 @@ The datamodel can be found at `./database/datamodel.graphql`. By the end of the
...
-- [ ] Update and Delete functionality
+- [ ] CRUD functionality for WineTasters
+- [ ] Include a small Login Component to start supporting Users in the app.
 - [ ] Review expanded with fields such as: predicted price, suggested pairing, or predicted year
-- [ ] WineTaster expanded to include other user profile fields
-- [ ] Wines have an attached image (a placeholder image is found in `./src/assets`)
-- [ ] TastingSessions have a date
+- [ ] Include another section in the app to search reviews and filter them based on different parameters (such as best/worst rated, date, price).
+- [ ] Support the ability to use enviroment variables. Both in the Front and Back of the app.
+
```
Comment (danaszova) [inline README.md:37]: Front end updates look good, but there was the bug we saw when you demonstrated the work to us yesterday. We could not see the taster being added I think is what it was. Could we include the bug - and instructions to fix it in the ReadMe?

**Review by danaszova — CHANGES_REQUESTED** [review]
Good start - Left some comments

File: README.md (hunk 2 above)
Comment (ntaboada) [reply README.md:14]: I forgot about that. I will add it as another task for the challenge :)

File: package.json (hunk 3 above)
Comment (ntaboada) [reply package.json]: i don't think so. This is being used for the modals

File: README.md (hunk 2 above)
Comment (justinkaseman) [reply README.md:14]: I would also do a quick QA while you have it up and running to check for any other major bugs. If any are found just mention them in the README.

File: package.json (hunk 3 above)
Comment (justinkaseman) [reply package.json]: But if this application reached a production employment this package would not be installed. Would that cause breaking/unintended side effects?

Comment (ntaboada) [reply package.json]: in that case you are right. So i will add this change too.

File: README.md (hunk 4 above)
Comment (ntaboada) [reply README.md:37]: I remembered that the thing that supposed to be a bug wasn't really a bug. I just didn't worked on showing the list of WineTasters on that page. I can add that as a feature too. Thoughts?

Comment (danaszova) [reply README.md:37]: Sure - as longs as its well described

**Pull request closed without merging**